java-client/src/main/java/org/apache/pegasus/client/PegasusTable.java

```diff
@@ -1672,14 +1731,18 @@ public int batchDel2(List<Pair<byte[], byte[]>> keys, List<PException> results,
   @Override
   public void multiDel(byte[] hashKey, List<byte[]> sortKeys, int timeout) throws PException {
     if (timeout <= 0) timeout = defaultTimeout;
-    int count = sortKeys == null ? 0 : sortKeys.size();
+    int count = sortKeys == null ? -1 : sortKeys.size();
```

Comment:
Why would you update count from 0 into -1 when sortKeys is null?

Reply:
There are nothing special meaning,just to keep consistent with other codes and -1 meaning invalid.